doc/release-notes/iceoryx2-unreleased.md

```diff
@@ -458,3 +461,22 @@
     let node = NodeBuilder::new().create::<ipc::Service>()?;
     let node = NodeBuilder::new().create::<local::Service>()?;
     ```
+
+20. `SampleMutUninit` without `send` as replacement for `SampleMut<MaybeUninit<Payload>>`
+
+    ```rust
+    // old
+    let sample: SampleMut<zero_copy::Service, MaybeUninit<u64>> = publisher.loan_uninit()?;
+    let sample = sample.write_payload(123);
+    let sample.send()?;
+
+    // new
+    let sample: SampleMutUninit<zero_copy::Service, MaybeUninit<u64>> = publisher.loan_uninit()?;
```

Comment:
Shouldn't this be just
```suggestion
    let sample: SampleMutUninit<zero_copy::Service, u64> = publisher.loan_uninit()?;
```

Reply:
Nope, `MaybeUninit` is still required. The reason is that `Payload: ?Sized` but you cannot put a type into a maybe uninit where the size is not known at compile time.
Internally, the `SampleMutUninit` just holds a `SampleMut<..., Payload>`. If I pack this there into a `MaybeUninit` it fails to compile for slices.

Reply:
I'll have a look at this later today. If you want to merge this sooner, we can also do it and I create a new PR if I find a solution